README.md

```diff
@@ -53,12 +53,36 @@ This runs [Prettier][] to automatically format the entire codebase.
 
 This runs [Prettier][] on only your changed files. This is intended for development.
 
-### `yarn start`
```

Comment:
I don't see `yarn start` documented. Can you add that?

Reply:
I don't really know how to document it yet because it is more an internal command now, i.e. it compiles and starts the app in production mode (but not for local usage). Maybe we should actually remove it for now?

Reply:
If it's for production then we definitely need to document it.

Reply:
Yep, but it is not really ready yet. There is no `Dockerfile`, etc. I'd rather file an issue to create those commands when we'll have hosted envs.